### DISCUSSION TITLE
关于Doris  客户端的咨询

### DISCUSSION THREAD

**Discussion by BugZheng** [apache/doris#11847] [A - General / Q&A]
请问Doris 官方有没有提供编程语言客户端SDK, 或者有没有推荐使用的编程语言及推荐理由？

**Comment by dataalive** [comment]
Doris兼容Mysql协议，对语言没有偏好。对于导入支持http接口，各种语言也都OK。

**Comment by BugZheng** [comment]
nice , thanks

**Comment by fwchitu** [comment]
可以借助mysql的库，工具来连接doris使用
